nx-X11/programs/Xserver/hw/nxagent/Composite.c

```diff
@@ -24,18 +24,8 @@
 #include "Options.h"
 #include "Windows.h"
 
-#undef  NXAGENT_UPGRADE
-
-#ifndef NXAGENT_UPGRADE
-
-#include "NXcomposite.h"
```

Comment:
Uhm... wait. This is undefining `NXAGENT_UPGRADE` and then pulling in `NXcomposite.h` if it's undefined.

So the old code does basically only ever `#include "NXcomposite.h"`.

`nx-X11/programs/Xserver/hw/nxagent/NXcomposite.h` is somewhat functionally equivalent to `nx-X11/lib/Xcomposite/Xcomposite.h`, although the NX version does include one more "NX" file and defined `XserverRegion` prior to that.

Nothing seems currently to include `Xcomposite.h` and `composite.h` is probably not enough.